cytominer_eval/operations/util.py

```diff
@@ -131,3 +133,87 @@ def get_grit_entry(df: pd.DataFrame, col: str) -> str:
         len(entries.unique()) == 1
     ), "grit is calculated for each perturbation independently"
     return str(list(entries)[0])
+
+
+class DistributionEstimator:
```

Comment:
This class would be suitable for other distribution-based distances but for now I agree `MahalanobisEstimator` is more explicit (and this can always be reverted later if needed).